adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -217,6 +217,8 @@ private AuthenticationConstants() { }
         static final String EXT_EXPIRES_IN = "ext_expires_in";
 
         static final String CLAIMS = "claims";
+
+        static final String CLOUD_INSTANCE_NAME = "cloud_instance_name";
```

Comment:
please do update to the latest parameter name. It is available in test slice now.